context/app/static/js/components/workspaces/constants.ts

```diff
@@ -1,5 +1,11 @@
+/* Workspace job types */
+export const JUPYTER_LAB_JOB_TYPE = 'jupyter_lab';
+export const JUPYTER_LAB_R_JOB_TYPE = 'jupyter_lab_r';
+export const JUPYTER_LAB_GPU_JOB_TYPE = 'jupyter_lab_gpu_common_packages';
+export const JUPYTER_LAB_NON_GPU_JOB_TYPE = 'jupyter_lab_non_gpu_common_packages';
```

Comment:
This setup makes sense to me, but there may be some naming conflicts to resolve with the constants I created for these in #3540.